### PR TITLE
updates Web Animations to match modern spec

### PR DESCRIPTION
**Pull request by samthor**
@ebidel - this should solve #1332

### CODE REVIEW

**Comment by samthor** [comment]
@adanilo if you're curious!

File: 0.5/platform/web-animations.md [hunk 1]
```diff
@@ -55,36 +53,32 @@ a `<div>` over 0.5 seconds. The animation alternates producing a pulsing effect.
...
 
-The Web Animations model is a description of an engine for animation content on the web. The engine is sufficiently powerful to support CSS Transitions, CSS Animations and SVG Animations.
+The Web Animations model is a description of an engine for animation content on the web. In browsers like Chrome, the engine is sufficiently powerful to support CSS animations.
 
 Web Animations also exposes a JS API to the model. This API defines a number of
 new interfaces that are exposed to JavaScript. We'll go through some of the more
-important ones here: Animations, AnimationEffects, TimingDictionaries, TimingGroups, and AnimationPlayers.
-
-An `Animation` object defines a single animation effect that applies to a single element target. For example:
+important ones here: `KeyframeEffect`, `AnimationEffectReadOnly`, `AnimationEffectTiming`, and `Animation`.
```
Comment (ebidel) [inline 0.5/platform/web-animations.md]: no more demo?

File: 0.5/platform/web-animations.md [hunk 2]
```diff
@@ -55,36 +53,32 @@ a `<div>` over 0.5 seconds. The animation alternates producing a pulsing effect.
...
+        [{left: '0px'}, {left: '100px'}], 2000);
 
-An `AnimationEffect` object controls which CSS properties and SVG attributes are
-modified by an animation, and the values that those properties and attributes
-vary between. AnimationEffect objects also control whether the effect replaces
-or adds to the underlying value.
+Here, the target element's "left" CSS property is modified smoothly from `0px` to `100px` over 2 seconds. It's a subclass of `AnimationEffectReadOnly`, which provides timing configuration.
 
-There are three major kinds of effects: `KeyframeEffect`, `MotionPathEffect`, and `EffectCallback`.
+Note! Animating the "left" property is good for simple sites, but it will cause the browser to run layout on every frame. Regardless of your approach to animation, try to stick to animating "tranform" and "opacity" for the greatest performance.
```
Comment (ebidel) [inline 0.5/platform/web-animations.md]: we should just show this as the example :) or update the example to use opacity or something else.

**Comment by ebidel** [comment]
@arthurevans we should probably move this into the 1.0 docs now that it's up to date. Where would be a good places?

**Comment by arthurevans** [comment]
My understanding is that the web animations polyfill is no longer shipping with Polymer, so it seems like the connection is mostly historical.

I think it's used in neon-animation, but shouldn't this kind of doc be on the web animations repo?

**Comment by ebidel** [comment]
We haven't shipped the WA polyfill for a long time, even in 0.5 days. It would still be nice to have a place for ancillary new technologies like this on the site. WA is used by the neon-elements: https://elements.polymer-project.org/guides/using-neon-animations and we don't have other info on the API anywhere else.

File: 0.5/platform/web-animations.md (hunk 2 above)
Comment (samthor) [inline 0.5/platform/web-animations.md]: I've updated the example, and simplified this paragraph - linking to Web Fundamentals (which we do a few places over the Polymer docs).

**Comment by samthor** [comment]
@ebidel @arthurevans I'd like to get these changes in for now, then we can decide where to move it :)

**Comment by ebidel** [comment]
:+1:

**Comment by arthurevans** [comment]
Cool. Thanks @samthor! @ebidel, I'll make a note to bring this up in docs meeting. Another alternative would be for this info to be on, say, MDN, which right now is kind of lacking:

https://developer.mozilla.org/en-US/docs/Web/API/Web_Animations_API

**Comment by samthor** [comment]
I'd love to contribute to MDN! I'm not quite sure how to do that, though.

**Pull request merged**